app/src/components/Footer/components/Nav.tsx

```diff
@@ -39,6 +39,10 @@ const Nav: React.FC = () => {
 
 const StyledNav = styled.nav`
   display: flex;
+  background-color: ${(props) => props.theme.color.grey[800]};
+  border-radius: 10px;
+  opacity: 90%;
+  pointer-events: all;
```

Comment:
alphabetize

Reply:
i wish this was a lint feature... hmmm tool idea